ironfish/src/blockchain/blockchain.ts

```diff
@@ -1120,19 +1120,15 @@ export class Blockchain {
   }
 
   /**
-   * Iterates through transactions, starting from fromHash or the genesis block,
+   * Iterates through block headers, starting from fromHash or the genesis block,
    * to toHash or the heaviest head.
    */
-  async *iterateTransactions(
+  async *iterateBlockHeaders(
```

Comment:
Changed this to just iterate blockHeaders since this function was only used by scanTransactions, and I wanted to iterate blockHeaders so we could track the progress without lots of weird logic by guessing on the sequence returned by `iterateBlockTransactions`